pkg/couchdb/ui/src/Button/Button.stories.svelte

```diff
@@ -0,0 +1,117 @@
+<script lang="ts">
+	import { Meta, Story, Template } from '@storybook/addon-svelte-csf';
+
+	import { Mail } from '@librocco/svg';
+
```

Comment:
Delete the entire `src` folder (well...you can leave the folder, but delete the contents)